src/cls/rgw/cls_rgw.cc

```diff
@@ -1398,7 +1398,6 @@ class BIVerObjEntry {
                                                                               keep separate instance entry for the delete markers */
 
     if (ret < 0) {
-      CLS_LOG(0, "ERROR: read_key_entry() idx=%s ret=%d", instance_idx.c_str(), ret);
```

Comment:
When deleting a versioned obj, it generates a rand instance in delete_obj for delete marker, but name+instance does not commit to disk right now in this func. So  every time deleting a versioned obj, instance_id is initialized as delete marker in read_key_entry func, but without instance_entry, as a result, this log printed as -ENOENT.

Reply:
But instance_entry is initialized in init_as_delete_marker finally, so i do not think this is a bug.

Reply:
For normal writing op, if instance_idx does not initialized as a real data error, it just return and leaves initialized as false. It will init again before write_entries, it is proper in my operspective.